registrations.md

```diff
@@ -0,0 +1,52 @@
+## Registrations
+For nodes and users to interact with the Obscuro network they must register via the Network Management contract.
```

Comment:
Anyone wishing to join the Obsuro network as a Node must first check their hardware is compatible with the latest Attestation Constraints. Then they must download and install the latest software, and then they must perform the steps detailed below.